lib/runner.js

```diff
@@ -386,6 +386,15 @@ Runner.prototype.hook = function(name, fn) {
       }
       if (err) {
         if (err instanceof Pending) {
+          if (name === HOOK_TYPE_AFTER_EACH || name === HOOK_TYPE_AFTER_ALL) {
+            utils.deprecate(
+              util.format(
+                "skipping a test within an '%s' hook is DEPRECATED and will throw an exception in a future version of Mocha. " +
```

Comment:
No need to quote internal to format string. Extraneous space at end.

Reply:
` ' [...] of Mocha.  ' + 'Use a return [...]' `  This space is set intentionally.

internal quotes: I prefer my code style.